mongodbatlas/data_source_mongodbatlas_alert_configuration.go

```diff
@@ -504,3 +507,178 @@ func convertNotificationToCtyValues(notification *matlas.Notification) map[strin
 
 	return values
 }
+
```

Comment:
these are methods that I moved from old resource implementation as they are used in the data source, will be removed when data source is migrated as well.